tests/common/fixtures/advanced_reboot.py

```diff
@@ -71,6 +81,13 @@ def __extractTestParam(self):
         self.readyTimeout = self.request.config.getoption("--ready_timeout")
         self.replaceFastRebootScript = self.request.config.getoption("--replace_fast_reboot_script")
 
+        # Set default reboot limit if it is not given
```

Comment:
this effectively eliminate the ability to control the reboot limit using command line args

Reply:
These lines will only set a value if no value is given by command line args. If there is a value given, `self.rebootLimit` would not be `None` and it would not be overwritten with the value here. Therefore, this should not affect the ability to control the reboot limit using command line args.